pennylane/devices/default_qubit_jax.py

```diff
@@ -0,0 +1,237 @@
+    array([-2.2526717e-01 -1.0086454e+00  1.3877788e-17])
+
+    There are a couple of things to keep in mind when using the ``"backprop"``
+    differentiation method for QNodes:
+
+    * You must use the ``"jax"`` interface for classical backpropagation, as JAX is
+      used as the device backend.
+
+    .. UsageDetails::
+
+        JAX does randomness in a special way when compared to NumPy, in that all randomness needs to
+        be seeded. While we handle this for you automatically in op-by-op mode, when using ``jax.jit``,
+        the automatically generated seed gets constantant compiled.
+
+        Example:
+
+        .. code-block:: python
+
+            dev = qml.device("default.qubit.jax", wires=1)
+
+            @jax.jit
+            @qml.qnode(dev, interface="jax", diff_method="backprop")
+            def circuit():
+                qml.Hadamard(0)
+                return qml.sample(qml.PauliZ(wires=0))
+
+            a = circuit()
+            b = circuit() # Bad! b will be the exact same samples as a.
+
+
+        To fix this, you should wrap your qnode in another function that takes a PRNGKey, and pass
+        that in during your device construction.
+
+        .. code-block:: python
+
+            @jax.jit
+            def keyed_circuit(key):
+                dev = qml.device("default.qubit.jax", interface="jax", prng_key=key)
+                @qml.qnode(dev, interface="jax", diff_method="backprop")
+                def circuit():
+                    qml.Hadamard(0)
+                    return qml.sample(qml.PauliZ(wires=0))
+                return circuit()
+
+            key1 = jax.random.PRNGKey(0)
+            key2 = jax.random.PRNGKey(1)
+            a = keyed_circuit(key1)
+            b = keyed_circuit(key2) # b will be different samples now.
+
+        Check out out the `JAX random documentation <https://jax.readthedocs.io/en/latest/jax.random.html>`__
+        for more information.
+
+    Args:
+        wires (int): The number of wires to initialize the device with.
+        shots (int): How many times the circuit should be evaluated (or sampled) to estimate
+            the expectation values. Defaults to 1000 if not specified.
+            If ``analytic == True``, then the number of shots is ignored
+            in the calculation of expectation values and variances, and only controls the number
+            of samples returned by ``sample``.
+        analytic (bool): Indicates if the device should calculate expectations
+            and variances analytically. In non-analytic mode, the ``diff_method="backprop"``
+            QNode differentiation method is not supported and it is recommended to consider
+            switching device to ``default.qubit`` and using ``diff_method="parameter-shift"``.
+        prng_key (Optional[jax.random.PRNGKey]): An optional jax.random.PRNGKey. This is the key to the
+            pseudo random number generator. If None, a random key will be generated. Default is None.

```

Comment:
```suggestion
        prng_key (Optional[jax.random.PRNGKey]): An optional ``jax.random.PRNGKey``. This is the key to the
            pseudo random number generator. If None, a random key will be generated.
```
(No need to include the default here, since Sphinx will automatically include it in the argument description)